src/renderer/state.ts

```diff
@@ -157,6 +165,21 @@ export class AppState {
   private readonly defaultVersion: string;
   public appData: string;
 
+  // Populating versions in fiddle-core
+  // TODO(aryanshridhar): Docs recommends not to use this in production!
+  public baseVersions: BaseVersions = new BaseVersions(getElectronVersions());
+
+  // For managing downloads and versions for electron
+  public installer: Installer = new Installer({
+    electronDownloads: ELECTRON_DOWNLOAD_PATH,
+    electronInstall: ELECTRON_INSTALL_PATH,
+  });
+
+  public runner: Promise<Runner> = Runner.create({
```

Comment:
This `runner` variable can be confused as an instance of [`Runner`](https://github.com/electron/fiddle/blob/440a94b4f833b077bf576022960d4fe737f04382/src/renderer/runner.ts#L30) class. Open to suggestions for better naming for this variable :) (Probably, `coreRunner`?)

Reply:
Depending on the functionality of the other existing class, something like `versionRunner` may work